pages/applications/index.vue

```diff
@@ -1,98 +1,123 @@
-    <p class="page-description">
-      If you feel that your organization or project is a good match with the Vatterott Foundation’s funding priorities, you may apply for a grant by downloading and submitting the following documents. Please <a href="mailto:info@vatterottfoundation.org">email</a> us if you have any questions.
-    </p>
+  <div>
+    <div class="container">
+      <h1 class="page-title">{{ title }}</h1>
+      <p class="page-description">
+        {{ subtitle }}
+      </p>
+      <div v-html="html" />
+      <div>
+        <h2>{{ appdescriptions.header2 }}</h2>
+
+        <div class="page-description" style="text-align: center;">
+           {{ appdescriptions.goodmatchA }}
+        <a href="mailto:info@vatterottfoundation.org">{{ appdescriptions.emailtextlink }}</a>
+            {{ appdescriptions.goodmatchB }}
+        </div>
+      </div>
+
     <fieldset>
-      <legend class="section-description">&nbsp;Please submit the following documents:&nbsp;</legend>
-      <ul class="checkboxes">
-        <li>
-          <input id="application" type="checkbox" />
-          <label for="application">Application Form (please <a href="/public/VF_Application_Form.docx">use attached format</a> which is specific for the Vatterott Foundation)</label>
-        </li>
-        <li>
-          <input id="project-budget" type="checkbox" />
-          <label for="project-budget">Project Budget Template Form (<a href="/public/VF_Budget_Template_Form.xls">download</a>)</label>
-        </li>
-        <li>
-          <input id="current-budget" type="checkbox" />
-          <label for="current-budget">Current Organizational Budget</label>
-        </li>
-        <li>
-          <input id="current-budget" type="checkbox" />
-          <label for="current-budget">IRS Letter of Determination</label>
-        </li>
-      </ul>
+        <legend class="section-description">
+          &nbsp;{{ documents.sectionlabel }}&nbsp;
+        </legend>
+        <ul class="checkboxes">
+          <li>
+            <input type="checkbox">
+            <label for="application"
+              >{{ documents.applicationlabelA }}
+              <a href="/public/VF_Application_Form.docx">{{
+                documents.applicationlink
+              }}</a>
+              {{ documents.applicationlabelB }}</label
+            >
+          </li>
+          <li>
+            <input type="checkbox">
+            <label for="project-budget"
+              >{{ documents.budgetlabel
+              }}<a href="/public/VF_Budget_Template_Form.xls">{{
+                documents.budgetlink
+              }}</a></label
+            >
+          </li>
+          <li>
+            <input type="checkbox">
+            <label for="current-budget">{{ documents.currentbudget }}</label>
+          </li>
+          <li>
+            <input type="checkbox">
+            <label for="current-budget">{{ documents.irsletter }}</label>
+          </li>
```

Comment:
Could we have a list added to the Markdown for this page? Authors can add and remove required documents. Just two fields: Label and an optional link to a template file. Then we can loop through and display those dynamically here.

Reply:
I tried doing that at first, but will explore it again.

Reply:
Done!